Hogajama/configuration/local_configuration/06_deploy.cli

```diff
@@ -0,0 +1,3 @@
+deploy --force ../hogajama-rs-0.0.1-SNAPSHOT.war
+deploy --force ../hogajama-frontend-0.0.1-SNAPSHOT.war
+deploy --force ../hogajama-angular-frontend-0.0.1-SNAPSHOT.war
```

Comment:
würde hier mMn Sinn ergeben, wenn die deployment-pakete dynamisch gefunden werden. Ansonsten muss bei jeder Versionsänderung (ja, in Hogarama eher nicht wahrscheinlich, aber ansonsten) auch das Skript gewartet werden

Reply:
Das stimmt grundsätzlich, man muss dabei aber in bisschen aufpassen, da in ../ nicht nur die Deployments von Hogajama drinnen liegen, sondern z.B. auch vom Mock-GUI. Daher sollte man ein bisschen darüber nachdenken bevor man das umsetzt.